flink-cdc-connect/flink-cdc-source-connectors/flink-connector-postgres-cdc/src/test/java/com/ververica/cdc/connectors/postgres/source/PostgresSourceITCase.java

```diff
@@ -254,6 +278,8 @@ public void testDebeziumSlotDropOnStop() throws Exception {
                 tableResult, FailoverType.JM, FailoverPhase.STREAM, new String[] {"customers"});
 
         tableResult.getJobClient().get().cancel().get();
+        Thread.sleep(1000L);
+        customDatabase.removeSlot(slotName);
```

Comment:
It is better to use `try ... finally ...`.

Reply:
Add it now.